tests/test_pytest.py

```diff
@@ -219,11 +217,15 @@ def collect_names(code):
                         # IMPORT_FROM(_Test__submodule11c)
                         # STORE_NAME(_Test__subc11)
 
+                        if instruction.opname=="LOAD_ATTR" and before is not None  and before.opname == "IMPORT_NAME":
```

Comment:
```suggestion
                        if instruction.opname == "LOAD_ATTR" and before is not None and before.opname == "IMPORT_NAME":
```

we really need proper tooling in this repo like ruff